pandas/_libs/tslibs/np_datetime.pyx

```diff
@@ -37,6 +37,12 @@ cdef extern from "src/datetime/np_datetime_strings.h":
                                 npy_datetimestruct *out,
                                 int *out_local, int *out_tzoffset)
 
+# ----------------------------------------------------------------------
+# time constants
+
+cdef int64_t DAY_S = 86400
```

Comment:
let's write this out to DAY_SECONDS

Reply:
I think the place for these may be ccalendar